man/scrot.txt

```diff
@@ -2,128 +2,135 @@ NAME
   scrot - command line screen capture utility
 
 SYNOPSIS
-  scrot [options] [file]
+  scrot [-bcDfhkmopsuvz] [-a X,Y,W,H] [-C CMD] [-d SEC] [-e CMD] [-l STYLE]
```

Comment:
The `-C` option denotes a class name not a command. Maybe `-C [NAME]`?

Reply:
Yes, that was a bad oversight.